wordpress-mobile/WordPress-iOS
# Issues/3856 use Merryweather for content font

Closes #3856 

This PR adopts the Merriweather font for "content" areas in the app. These include:
- Reader post cards
- Reader post detail
- Reader comments
- Post list cards
- Page list cells
- Comment detail
- Comment notifications detail
- The editor's visual mode
- The title in the editor's HTML mode.

UI elements that are not considered content, (headers, button labels, etc.) continue to use Open Sans. 

To test:
- Open each of the above features and confirm that content now appears in the Merriweather font.  
- Make sure that varying rich text content (links, bold, italics, blockquotes) in comments show Merriweather properly.

I'd like to ping a few people for review so feature experts can have an extra critical eye.
@oguzkocer, would you be up for a general review?
@jleandroperez, would you check we didn't miss anything in comments and comment notifications? 
@bummytime, would you mind double checking the editor in the live app? 

Needs Review: @oguzkocer @jleandroperez  @bummytime
